### PR TITLE
Imported character sheets now render on the proper language for the u…

### PR DESCRIPTION
**Pull request by spitfire305**
…ser.

### CODE REVIEW

File: app/Models/PluginVersion.php [hunk 1]
```diff
@@ -143,13 +154,31 @@ protected function renderBlade(Entity $entity)
 
         $html = str_replace([
             '@php', '@dd', '@inject', '@yield', '@section', '@auth', '@guest', '@env', '@once', '@push', '@csrf',
-            '@include', '\Illuminate\\',
+            '@include', '\Illuminate\\', "@i18n('"
```
Comment (ilestis) [inline app/Models/PluginVersion.php:157]: This block is for removing blacklisted functions, so wrong place to add this :)

File: app/Models/PluginVersion.php [hunk 2]
```diff
@@ -143,13 +154,31 @@ protected function renderBlade(Entity $entity)
...
-            '', '', '', '', '', '', '', '', '', '', ''
+            '', '', '', '', '', '', '', '', '', '', '', '', '', "trans('*."
         ], $html);
         $html = preg_replace('`dd\((.*?)\)`i', '', $html);
         $html = preg_replace('`config\((.*?)\)`i', '', $html);
+        $locale = auth()->user()->locale;
+        $lines = [];
+
+        //Replace translation keys of existing translations
+        foreach ($this->getTranslationsAttribute() as $translation) {
```
Comment (ilestis) [inline app/Models/PluginVersion.php:167]: You can replace these whole two blocks with a single `preg_replace_content` like it's done for `@liveAttribute` and 3 lines :-)

**Review by ilestis — CHANGES_REQUESTED** [review]
Can be refactored into 3 lines of code

File: app/Models/PluginVersion.php (hunk 2 above)
Comment (spitfire305) [reply app/Models/PluginVersion.php:176]: `preg_replace_callback` doesnt seem to be working, do I leave the current implementation while its fixed?

**Pull request merged**